src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -339,6 +339,62 @@ image:compose-configuration/page-tree-base.png[width=49%]
 image:compose-configuration/page-tree-nest.png[width=49%]
 image:compose-configuration/page-tree-nest-view.png[]
 
+==== Expand sub-pages
+
+By default, in order to expand a sub-page in the sidebar, you have to either visit the page holding the sub-page or expand the sub-page using the arrow down button.
+
+To always have a sub-page expanded, navigate to the menu:admin panel[pages], and edit the page you wish to have its sub-page expanded.
```

Comment:
I think we can drop the whole how to get there/where it's located and just tell what option in the page edit it is